vm-virtio/src/lib.rs

```diff
@@ -61,6 +62,7 @@ impl From<u32> for VirtioDeviceType {
             24 => VirtioDeviceType::TYPE_MEM,
             26 => VirtioDeviceType::TYPE_FS,
             27 => VirtioDeviceType::TYPE_PMEM,
+            28 => VirtioDeviceType::TYPE_WATCHDOG,
```

Comment:
There's a mismatch here. Should be `35`.

Reply:
Fixed.